lib/views/pages/profile/components/profile_badge.dart

```diff
@@ -0,0 +1,37 @@
+import "package:flutter/material.dart";
+
+/// Info card for the badges in profile
+class ProfileBadge extends StatelessWidget {
+  static const badgeKey = Key("profileBadge");
+
+  static const _badgeWidth = 54.0;
```

Comment:
By curiosity, does this adapt well to all different phone sizes? As it is a hard coded value.

Reply:
Yeah it is basically like an icon size. I don't think it's a problem to have fixed values for those, it's like padding or things like that.